frontend/src/components/LoginDialog.jsx

```diff
@@ -0,0 +1,72 @@
+import React from "react";
+import Button from "@material-ui/core/Button";
+import TextField from "@material-ui/core/TextField";
+import BackendService from "../services/BackendService"
+
+import Dialog from '@material-ui/core/Dialog';
+import DialogActions from '@material-ui/core/DialogActions';
+import DialogContent from '@material-ui/core/DialogContent';
+import DialogContentText from '@material-ui/core/DialogContentText';
+import DialogTitle from '@material-ui/core/DialogTitle';
+import Cookies from 'universal-cookie';
+
+export default function LoginDialog() {
+  const [open, setOpen] = React.useState(false);
+  const [user, setUser] = React.useState('');
+  const [pwd, setPwd] = React.useState('');
```

Comment:
No need to shorten the name, [password, setPassword] is still short and looks more clear imo